modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -1,14 +1,49 @@
-[id="anchor-and-file-names"]
-= Anchor Names and File Names
+[id="module_anchor-and-file-names-concept"]
+= File Names and Anchors
 
-To optimize modular documentation, follow these guidelines for naming module anchors and files:
+To optimize modular documentation, follow these guidelines for naming files and creating anchors:
 
-Anchor names:: Provide an anchor in the format `+++[id="anchor-name_{context}"]+++` for every module so that it can be identified by Asciidoctor when reused or cross-referenced. `+++{context}+++` is a variable whose value you define in the assembly. Give the anchor the same or similar name as the module heading. Separate the words in the anchor with hyphens and use an underscore to separate the `+++{context}+++` part:
-+
+.File names
+Assembly and module file names should accurately and closely reflect the title of the assembly or module. File names should have the format `prefix-filename.adoc` or `prefix_filename.adoc` where `prefix` is one of the following module prefixes:
+
+* `con`: Concept module prefix
+* `proc`: Procedure module prefix
+* `ref`: Reference module prefix
+* `assembly`: Assembly module prefix
+
+.Examples
+* `con-guided-decision-tables.adoc`  (Concept module)
+* `proc-creating-guided-decision-tables.adoc`  (Procedure module for creating)
+* `proc-editing-guided-decision-tables.adoc`  (Procedure module for editing)
+* `ref-guided-decision-table-examples.adoc`  (Reference module with examples)
+* `ref-guided-decision-table-columns.adoc`  (Reference module with column types)
+* `assembly-designing-guided-decision-tables.adoc`  (Assembly of guided decision table modules)
+
+
+[NOTE]
+====
+Do not include special characters in file names. Ensure that all members of your team use the same file naming conventions.
+====
+
+These file naming guidelines are optional but highly recommended. However, if your team does not include the module prefixes in file names followed by either a hyphen (-) or an underscore (_), you must include one of the following variables in each concept, procedure, and reference module:
```

Comment:
Maybe we want to make it a little stronger than optional. How about this:

>As an alternative to these file-naming conventions, if your team cannot include the module prefixes in file names followed by either a hyphen (-) or an underscore (_), you can include one of the following variables in each concept, procedure, and reference module:

Reply:
@sterobin Sorry Stetson. I think this should be diluted and both options presented as an equal candidate. If tooling is not forcing our hand on this, then both options are equal.

Reply:
This section is about file naming guidelines and anchor names. We have noted the requirement to specify the module type if a group chooses a different file naming convention. I suggest we leave the wording as it is.

Reply:
@vikram-redhat Understood, thanks.